tests/erc7432/approval-handler.test.ts

```diff
@@ -0,0 +1,55 @@
+import { assert, describe, test, clearStore, afterEach } from 'matchstick-as'
+import { createNewRoleApprovalForAllEvent } from '../helpers/events'
+import { validateRoleApproval, createMockRoleApproval } from '../helpers/entities'
+import { Addresses } from '../helpers/contants'
+import { handleRoleApprovalForAll } from '../../src/erc7432'
+
+const grantor = Addresses[0]
+const operator = Addresses[1]
+const tokenAddress = Addresses[2]
+
+describe('ERC-7432 RoleApprovalForAll Handler', () => {
+  afterEach(() => {
+    clearStore()
+  })
+
+  test('should not do anything when approval does not exist and is set to false', () => {
```

Comment:
We are following a [recommended test structure](https://thegraph.com/docs/en/developing/unit-testing-framework/#tests-structure-050) in our other repositories. Maybe we could updated this tests as well

```ts
describe("handlerName()", () => {
  describe("When entity exists", () => {
      ... test cases here
  })

  describe("When entity does not exists", () => {
      ... test cases here
  })
})
```